### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'clickhouseOptions.queryOptions')

### ISSUE THREAD

**Issue by bzed** [metrico/qryn#482]
Hi,

seems there was a bug in #361 -
```
54 |       return
55 |     }
56 |     const _queue = this.queue
57 |     this.queue = []
58 |     await rawRequest(this.statement, _queue.join('\n'), DATABASE_NAME(), { maxBodyLength: Infinity })
59 |   }
                                                                                     ^
TypeError: undefined is not an object (evaluating 'clickhouseOptions.queryOptions')
      at /app/lib/db/throttler.js:59:81
      at anonymous (native:1:1)
      at /app/lib/db/clickhouse.js:92:9
      at anonymous (native:1:1)
      at /app/lib/handlers/push.js:24:7
```

I think
https://github.com/metrico/qryn/blob/b4cda9e656f6720fe5ee7eaccb9dcd2c12e6ac82/lib/db/throttler.js#L3
should require clickhouse_options now?

Thanks,

Bernd

**Comment by akvlad** [comment]
@bzed apparently a bun problem. Will be fixed today. Please use the NodeJS version for now.

**Comment by akvlad** [comment]
@bzed everything fixed on 3.2.16 . Sorry for the inconvenience.

**Comment by lmangani** [comment]
Closing are resolved, please feel free to reopen if needed and thanks for the report!
